Refit/RequestBuilderImplementation.cs

```diff
@@ -445,7 +440,7 @@ void AddMultipartItem(MultipartFormDataContent multiPartContent, string fileName
                 var keyType = key.GetType();
                 var formattedKey = settings.UrlParameterFormatter.Format(key, keyType, keyType);
```

Comment:
Actually, I didn't see this before 😮 Apparently dictionary keys are formatted using `UrlParameterFormatter.Format`.

This accepts any `object` type whereas my impl normalizes (and requires) `string`.